packages/core/package.json

```diff
@@ -2,7 +2,10 @@
   "name": "@commons-ui/core",
   "version": "1.0.0",
   "dependencies": {
-    "@babel/runtime": "^7.8.3"
+    "@babel/runtime": "^7.8.3",
```

Comment:
We shouldn't depend on `@babel\runtime`, there is a build step that should transform our code down to ES5 or something.